### PR TITLE
Add data privacy page

### LINKED ISSUE #2826: /legal/data-privacy - build new 'data privacy' page

### PR DESCRIPTION
**Pull request by steverydz**
## Done

- Add new data privacy page
- Archive old privacy policy page
- Add redirects from old privacy policy page to new data privacy page
- Update cookie link in footer

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/legal/dataprivacy>
- Make sure page matches [the copy doc](https://docs.google.com/document/d/1AlOE2_-BqZNP4hiiuf8kZney8g1zvqAlDYl2u3X6m6I/edit#)
- Make sure <http://0.0.0.0:8001/legal/terms-and-policies/privacy-policy> redirects to <http://0.0.0.0:8001/legal/dataprivacy>
- Make sure footer contains a link to the data privacy page

## Issue / Card

Fixes #2826

### CODE REVIEW

**Comment by frankiegranato** [comment]
Looks great @steverydz

File: templates/legal/dataprivacy/index.html [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+        <li class="p-list__item"><strong>Functionality cookies</strong>. These are used to recognise you when you return to our website. This enables us to personalise our content for you, greet you by name and remember your preferences (for example, your choice of language or region).</li>
+        <li class="p-list__item"><strong>Targeting cookies</strong>. These cookies record your visit to our website, the pages you have visited and the links you have followed. We will use this information to make our website and the advertising displayed on it more relevant to your interests. We may also share this information with third parties for this purpose.</li>
+      </ul>
+      <p>You can find more information about the individual cookies we use and the purposes for which we use them in the table below:</p>
+      <table class="p-table">
```
Comment (Caleb-Ellis) [inline templates/legal/dataprivacy/index.html]: This table looks pretty tight. Maybe split it out into a col-12?

File: templates/legal/dataprivacy/index.html [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+          </tr>
+          <tr>
+            <td>LinkedIn</td>
+            <td><code>bcookie</code>, <code>lidc</code>, <code>BizoID</code>, <code>UserMatchHistory</code>, <code>lang</code>, <code>bscookie</code></td>
+            <td>
+              <p>LinkedIn tracking allows us to see how many people take action on our ads and which ad led to a conversion.</p>
+              <p><a class="p-link--external" href="https://www.linkedin.com/legal/privacy-policy">LinkedIn&rsquo;s privacy policy</a></p>
+            </td>
+          </tr>
+          <tr>
```
Comment (Caleb-Ellis) [inline templates/legal/dataprivacy/index.html]: Missing Twitter row

**Comment by steverydz** [comment]
@Caleb-Ellis Fixed

**Pull request merged**